Mathlib/RingTheory/Ideal/Norm/RelNorm.lean

```diff
@@ -177,6 +177,23 @@ theorem spanNorm_mul_of_bot_or_top (eq_bot_or_top : ∀ I : Ideal R, I = ⊥ ∨
   rw [hJ]
   exact le_top
 
+theorem spanNorm_le_comap (I : Ideal S) :
+    spanNorm R I ≤ comap (algebraMap R S) I := by
+  rw [spanNorm, Ideal.map, Ideal.span_le, ← Submodule.span_le]
+  intro x hx
+  induction hx using Submodule.span_induction with
+  | mem _ h =>
+      obtain ⟨x, hx, rfl⟩ := h
+      exact mem_comap.mpr <| mem_of_dvd _ (Algebra.dvd_algebraMap_intNorm_self _ _ x) hx
+  | zero => simp
+  | add _ _ _ _ hx hy => exact Submodule.add_mem _ hx hy
+  | smul _ _ _ hx => exact Submodule.smul_mem _ _ hx
+
+theorem bot_of_liesOver_bot (P : Ideal S) [h : P.LiesOver (⊥ : Ideal R)] :
```

Comment:
Is this [Ideal.under_bot](https://leanprover-community.github.io/mathlib4_docs/Mathlib/RingTheory/Ideal/Over.html#Ideal.under_bot)?

Reply:
Actually, this result already exists but with another formulation (and in much more generality), I just added the `Ideal.LiesOver` version in #28545